libraries/core-react/src/components/Menu/MenuSection.tsx

```diff
@@ -6,7 +6,7 @@ import { spacingsTemplate } from '../../utils'
 import { Divider } from '../Divider'
 import { Typography } from '../Typography'
 
-const ListItem = styled.li.attrs(() => ({
+const ListItem = styled.div.attrs(() => ({
```

Comment:
Suggest changing this to `Title` or `Header` something more descriptive as its a container for section name

Reply:
agreed, done 👍